tensorrt_llm/_torch/modules/fused_moe/quantization.py

```diff
@@ -224,9 +224,10 @@ def load_expert_weights_to_dst(
                     MoEWeightLoadingMode.VANILLA,
                     MoEWeightLoadingMode.W4A8_CUSTOM
             ]:
-                w1_weight = weights[f"{expert_id}.w1.weight"]
-                w3_weight = weights[f"{expert_id}.w3.weight"]
-                w2_weight = weights[f"{expert_id}.w2.weight"]
+                weight_name = "qweight" if f"{expert_id}.w1.qweight" in weights else "weight"
+                w1_weight = weights[f"{expert_id}.w1.{weight_name}"]
+                w3_weight = weights[f"{expert_id}.w3.{weight_name}"]
+                w2_weight = weights[f"{expert_id}.w2.{weight_name}"]
```

Comment:
For changes in `quantization.py`, we should add respective tests in 
`tests/unittest/_torch/modules/test_fused_moe.py::test_fused_moe_w4afp8` to avoid breaking this feature in the future